server/routes/applications/applicationDetailsRoutes.ts

```diff
@@ -35,18 +36,34 @@ export default function applicationDetailsRoutes({ auditService }: { auditServic
     asyncMiddleware(async (req: Request, res: Response) => {
       const { applicationData } = req.session
 
+      const applicationType = getApplicationType(req.session.applicationData?.type.apiValue)
```

Comment:
You've already pulled `applicationData` out on the line above so no need to reference `req.session`